assets/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from employees.models import Employee, BaseModel
+
+
+class Asset(BaseModel):
+    name = models.CharField(max_length=50)
+    asset_model = models.CharField(max_length=50)
+    type = models.CharField(max_length=50)
```

Comment:
change this column name form type to 'asset_type' as type is reserved word for other operations